### ISSUE TITLE
Is this plugin even required anymore

### ISSUE THREAD

**Issue by jrock2004** [mdehoog/ember-data-has-many-query#30]
I have upgraded my app to 2.18.2, is this no longer needed? Have they built this functionality into Ember itself? I am asking as there a deprecation warnings that we need to set babel version to 6.6

**Comment by gibso** [comment]
i dont think so, there are no such methods in the DS.ManyArray class https://www.emberjs.com/api/ember-data/3.5/classes/DS.ManyArray/methods
